### PR TITLE
Latest updates for VS Mac 2022

### LINKED ISSUE #24626: SQLite for dev section refers to startup.cs, needs update for 6.0
***EDIT by @Rick-Anderson*** Fix [VS-mac Scaffold the movie model](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-6.0&tabs=visual-studio-mac&source=docs#scaffold-the-movie-model)

There doesn't seem to be any autogenerated Startup.cs class following the VS for Mac guide. Could it be made less confusing?

It was also optional to omit this step with out any issues.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6719f08e-3bd7-dc1a-71df-f2ef9fbca9d8
* Version Independent ID: 7096fdb3-612e-9e00-bd0b-8ea4886a09ce
* Content: [Part 2, add a model](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-6.0&tabs=visual-studio-mac&source=docs)
* Content Source: [aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### LINKED ISSUE #25830: VS for Mac RC: Update Razor Pages Tutorial Series for latest changes
Update the Get started with Razor Pages tutorial series to reflect the latest changes that occurred in VS for Mac 2022 RC

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a25bf940-8977-fec4-461b-542a9f550fc8
* Version Independent ID: 5492c641-af15-64b8-beaa-ddf511be73e6
* Content: [Tutorial: Get started with Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/razor-pages-start.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/razor-pages-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### LINKED ISSUE #25878: Documentation for .NET 6.0 says "*when .NET 6 is released*"
From section **Fix the warning messages**:

> Ignore NU1603 package mismatch warnings, they will be fixed when .NET 6 is released.

This should be updated. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6719f08e-3bd7-dc1a-71df-f2ef9fbca9d8
* Version Independent ID: 7096fdb3-612e-9e00-bd0b-8ea4886a09ce
* Content: [Part 2, add a model](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by codemillmatt**
[Internal review of this tutorial series](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-6.0&branch=pr-en-us-25485&tabs=visual-studio)

Fixes #25830
Fixes #24626
Fixes #25878

### CODE REVIEW

**Comment by codemillmatt** [comment]
cc: @jongalloway, @jmatthiesen

**Comment by wadepickett** [comment]
@codemillmatt, I will jump in here to help, if that's OK.  Thanks for the updated content and screenshots!

**Comment by wadepickett** [comment]
@codemillmatt, The doc build indicated it was successful, but looking at the details, it appeared to have failed.  I will close and re-open to force it to build again.

**Comment by wadepickett** [comment]
Doc build successful now.

File: aspnetcore/tutorials/razor-pages/razor-pages-start.md [hunk 1]
```diff
@@ -156,7 +154,7 @@ The default browser launched with the following URL: [https://localhost:5001](ht
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-Select **Run** > **Start Debugging** to launch the app. Visual Studio for Mac launches a browser and navigates to `https://localhost:<port>`, where `<port>` is a randomly chosen port number.
+Select **Debug** > **Start Debugging** to launch the app. Visual Studio for Mac launches a browser and navigates to `https://localhost:<port>`, where `<port>` is a randomly chosen port number.
```
Comment (wadepickett) [inline aspnetcore/tutorials/razor-pages/razor-pages-start.md:157]:
```suggestion
Select **Debug** > **Start Debugging** to launch the app. Visual Studio for Mac launches a browser and navigates to `https://localhost:<port>`, where `<port>` is the port number randomly assigned at project creation and is set in `Properties/launchSettings.json`.
```

**Comment by wadepickett** [comment]
Testing with the VS for Mac RC I ran into a EF data migration issue where it was not completing correctly.  It may be due to a mismatch of EF package versions.  We have a new update however so I am testing that out.  The CRUD scaffolding now also  fixes the nullable issue for properties, so we no longer need to instruct folks to fix, nor tell them they will receive a nullable warning, but I can move that to another issue so this is not held up.  We have this problem in more than one tutorial now.

File: aspnetcore/tutorials/razor-pages/razor-pages-start.md [hunk 2]
```diff
@@ -46,8 +46,6 @@ At the end of this tutorial, you'll have a working Razor Pages web app that you'
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-We hope to have Visual Studio for Mac instructions soon.
```
Comment (jmatthiesen) [inline aspnetcore/tutorials/razor-pages/razor-pages-start.md:49]: ❤️

**Review by wadepickett — APPROVED** [review]
@codemillmatt, this looks good for the scope of what this is addressing and is ready to squash and merge once you take it out of "draft" status.

**Comment by wadepickett** [comment]
@codemillmatt, looks good for the scope you are hitting here, I tested through the set and approved.  OK if we take it out of draft and merge?

**Comment by codemillmatt** [comment]
good to go @wadepickett - thank you!

**Pull request merged**